### ISSUE TITLE
[Feature] Show file size instead of duration

### ISSUE THREAD

**Issue by raviwarrier** [advplyr/audiobookshelf-app#838]
![Screenshot_2023-08-13-00-17-19-01_e9782661401ed6f222682e1e02b74bd4.jpg](https://github.com/advplyr/audiobookshelf-app/assets/42061703/aa63f68c-3058-423d-8a09-82bf8eafa4b3)

In the above image, the duration is displayed twice. I understand that in this case it's so because the audiobook has only 1 file, but from the perspective of file(s), the size may be a better detail to display as it might help the user determine data consumption while streaming or downloading and help them decide if they can and/or should play/download the book, especially if they have low bandwidth (a large book may take longer time to download then they may have before losing/disconnecting the lan/internet connection), low space on their devices, etc.

**Comment by realmain** [comment]
I disagree on "Show file size instead of duration". The top section shows total duration, and the bottom section shows duration of each individual file.

IMO, in the top section, it should show total file size for sure. Do not need to list the size of each individual file in the bottom section, as when you download a book via the app, it downloads it all.

The web app does show both **Size** & **Duration** among others, but for mobile, it'd be too crowded.
